operatorcontrolplane/v1alpha1/0000_10-pod-network-connectivity-check.crd.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Don't we support v1 now?

Reply:
I did try it initially, but the crd  codegen seemed to ignore it and insisted on creating the v1beta1 version instead. I can look into it later (plus i would like to clean up those scripts anyway)

Reply:
@damemi this should work, shouldn't it?

Reply:
which codegen are you using, `controller-gen`? Our CRD gen doesn't touch this setting that I'm aware of (just the openapi spec) so I'm wondering what tool could be causing this

Reply:
controller-gen has a `crdVersions` flag, which defaults to v1beta1: https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/gen.go#L63-L71

Reply:
@sttts, @damemi 
The version of controller-gen we are using (`github.com/openshift/kubernetes-sigs-controller-tools v0.2.1-37-ga3cca5d`) seems to be hard coded to use v1beta1 when generating CRDs. 
https://github.com/openshift/kubernetes-sigs-controller-tools/blob/a3cca5d66f230ea5ca3554aef7e130750b841825/pkg/crd/spec.go#L25

Reply:
@sanchezl I see. No blocker here. @damemi we should rebase controller-gen